index.html

```diff
@@ -233,46 +250,57 @@ <h1 class="s">Skills</h1>
       </div>
     </ul>
     <!-- Footer Form Section -->
-
-    <section class="content-4 center">
-      <h1 class="clr-blue">
-        I'm always interested in hearing about new projects, so if you'd like to
-        chat please get in touch.
-      </h1>
-    </section>
-    <section class="center">
-      <form action="https://formspree.io/f/xeqwrjjn" method="POST">
-        <input
-          type="text"
-          class="in"
-          maxlength="30"
-          name="name"
-          placeholder="&nbsp;&nbsp;&nbsp;Full name"
-          required
-        />
-        <input
-          type="email"
-          class="in"
-          name="email"
-          placeholder="&nbsp;&nbsp;&nbsp;Email address"
-          required
-        />
-        <textarea
-          name="text"
-          id=""
-          cols="30"
-          rows="10"
-          maxlength="500"
-          required
-          placeholder="
+    <footer>
+      <div class="foot-flex">
+        <section class="content-4 center">
+          <h1 class="clr-blue">
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h1>
+        </section>
+        <section class="center">
+          <form action="https://formspree.io/f/xeqwrjjn" method="POST">
+            <input
+              type="text"
+              class="in"
+              maxlength="30"
+              name="name"
+              placeholder="&nbsp;&nbsp;&nbsp;Full name"
+              required
+            />
+            <br />
+            <input
+              type="email"
+              class="in"
+              name="email"
+              placeholder="&nbsp;&nbsp;&nbsp;Email address"
+              required
+            />
+            <br />
+            <textarea
+              name="text"
+              id=""
+              class="text-in"
+              cols="30"
+              rows="10"
+              maxlength="500"
+              required
+              placeholder="
         Hey I’d like to connect to..."
-        ></textarea>
-        <img class="foot" src="imgs/footer.png" alt="illustration" />
-        <br />
-        <button class="btn btn-2 btn-3" type="submit">Get in touch</button>
-        <p class="clr-blue">blesslinjerish@gmail.com</p>
+            ></textarea>
+            <img class="foot" src="imgs/footer.png" alt="illustration" />
+            <br />
```

Comment:
- [ ] Kindly remove this image as it's causing an overlay and position the background image correctly as shown in the design.
    - Your design: 
      ![gTiKbGs5jn](https://user-images.githubusercontent.com/104968236/228811703-8c29331f-8030-4d99-a1c9-47bfec4e353c.png)
   - Figma design:
     ![firefox_ZAtoH8icVm](https://user-images.githubusercontent.com/104968236/228811933-2cea79df-2b40-452f-83e8-3687f1a948d6.png)